discovery-dns/src/main/resources/reference.conf

```diff
@@ -2,14 +2,25 @@
 # Akka Service Discovery DNS Config                  #
 ######################################################
 
-akka.discovery {
+akka {
+  io {
+    dns {
+      resolver = "async-dns"
```

Comment:
Trying to override reference.conf value from another reference.conf will not work. I think it must be documented that this is required to define in application.conf (with runtime check that fails if it isn't)